bot/skills-loader.js

```diff
@@ -81,30 +82,45 @@ var SkillsLoader = function() {
     }
 
     function installPredefinedSkills(controller, middleware, devSkill, callback) {
```

Comment:
The third argument should be 'skill' only. I know this is not part of your change